libs/application/templates/driving-license/src/forms/prerequisites/sectionExternalData.ts

```diff
@@ -25,48 +25,7 @@ export const sectionExternalData = buildSubSection({
       id: 'approveExternalData',
       subTitle: m.externalDataSubTitle,
       checkboxLabel: m.externalDataAgreement,
-      dataProviders: [
```

Comment:
this seems a bit off, do you need to remove sections from another application?

Reply:
absolutly not, This was a mistake 
Thanks for catching this!